clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -347,3 +346,34 @@ export async function waitForDiagnosticsChange(file: string | vscode.Uri) {
 
   return result;
 }
+
+export async function triggerCompletionsAndWaitForResults() {
+  while (true) {
```

Comment:
Will it make sense to have a timeout here similar to waitFor function?

Reply:
There is a general timeout for all tests built in mocha, that fails the test if the timeout is reached, so this would just duplicate it, no?

Reply:
That's right... 👍